qtconsole/console_widget.py

```diff
@@ -1680,28 +1680,6 @@ def _flush_pending_stream(self):
             int(max(100, (time.time() - t) * 1000))
         )
 
-    def _format_as_columns(self, items, separator='  '):
-        """ Transform a list of strings into a single string with columns.
-
-        Parameters
-        ----------
-        items : sequence of strings
-            The strings to process.
-
-        separator : str, optional [default is two spaces]
-            The string that separates columns.
-
-        Returns
-        -------
-        The formatted string.
-        """
-        # Calculate the number of characters available.
-        width = self._control.document().textWidth()
-        char_width = self._get_font_width()
-        displaywidth = max(10, (width / char_width) - 1)
-
-        return columnize(items, separator, displaywidth)
-
```

Comment:
For git-log of qtconsole, last usage of that was 2012, and is the last place where 'seprator' of columnize is configurable.